packages/ava/__tests__/unit/data/statistics/index.test.ts

```diff
@@ -1,3 +1,7 @@
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
Have you tested whether these methods are consistent with the results of `stdlib` before

Reply:
The results of the lowess method (regression method) are close but not identical.
Other methods have the same result

Reply:
Could the insight demos work?

Reply:
yes.
before : 
![image](https://user-images.githubusercontent.com/71261480/224949909-ab657f95-08ae-4793-93c9-45ff00164dea.png)

after :
![image](https://user-images.githubusercontent.com/71261480/224950174-ef7451f3-bfd6-4a91-b872-3bcd046304e7.png)